docs/en/sql-reference/functions/array-functions.md

```diff
@@ -2157,6 +2157,41 @@ Result:
 └───────────────────────────────────────────────┘
 ```
 
+## arrayPrAUC
```

Comment:
The function seems some special case of `arrayAUC`. Why is it not called `arrayAUCPr` then?

Reply:
Technically, both ROC AUC and PR AUC are special cases of a more generic AUC. However, since the AUC for the ROC curve is more commonly used, ROC AUC is usually just called AUC. Sklearn's differentiates AUC, ROC AUC and PR AUC:

- [auc](https://scikit-learn.org/1.5/modules/generated/sklearn.metrics.auc.html) (AUC)
- [roc_auc_score](https://scikit-learn.org/1.5/modules/generated/sklearn.metrics.roc_auc_score.html) (ROC AUC)
- [average_precision_score](https://scikit-learn.org/1.5/modules/generated/sklearn.metrics.average_precision_score.html) (PR AUC)

Now, regarding `arrayAUCPr` or `arrayPrAUC`. Both PR AUC and AUC PR are used, and the same can be said about ROC AUC or AUC ROC. I think that AUC as a suffix is the more common form, but as long as we stick with one, it should work.